cypress/plugins/index.ts

```diff
@@ -0,0 +1,40 @@
+// ***********************************************************
+// This example plugins/index.js can be used to load plugins
+//
+// You can change the location of this file or turn off loading
+// the plugins file with the 'pluginsFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/plugins-guide
+// ***********************************************************
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
This is one way of solving the problem, and probably the easier one :slightly_smiling_face: Writing custom type definitions for an external package is easy enough in this case, so I would advise:

1. Creating a `d.ts` file somewhere in this TS project (e.g. `cypress/plugins/typings.d.ts`) with the following contents:

    ```ts
    // NOTE: preexisting types were not found
    declare module '@cypress/code-coverage/task' {
        const coverageTask: Cypress.PluginConfig;
        export default coverageTask;
    }
    ```

    Notice that I reused the `Cypress.PluginConfig` type from Cypress instead of writing the function itself, which I always encourage :slightly_smiling_face: 

2. Removing the comments here :smile: `coverageTask` will have the appropriate type
3. After making this change you will see errors below, when calling it. See comment related to that code for a solution

Reply:
Thanks for a ready solution - done